README.md

```diff
@@ -21,6 +21,7 @@ The plugin is enabled by default, and can be disabled by copying `user/plugins/d
 |----------------|---------|-------------------|--------------------------------------------------------------------------|
 | `enabled` | `true` | `true` or `false` | Enables or disables plugin entirely. |
 | `level` | 1 | (int) 0-10 | Initial level of folders to expand on load. |
+| `max_depth`  | 3 | (int) 1-10 | Deepest depth to which to generate the tree. |
```

Comment:
The logic here is sound, though you should note the Options-column as (int) 1-n - which really should be set for `level` as well, since we have not imposed any harm limits on this. Further, the Note-column should read "Depth at which to stop generating the tree." since this is a maximum limit, rather than a minimum limit.

Reply:
I've corrected the options column for both `max_depth` and `level`. Note that blueprints.yaml specified range widgets for the admin UI. I have changed them to textfields which validate to integers, with appropriate minimum values.

Reply:
I've also updated the description of the max_depth option.